samples/apps/bookinfo/bookinfo.yaml

```diff
@@ -156,7 +184,57 @@ spec:
         - containerPort: 9080
 ---
 ##################################################################################################
-# Productpage service
+# Mysql db services
+# credentials: root/password
+##################################################################################################
+apiVersion: v1
+kind: Secret
+metadata:
+  name: mysql-credentials
+type: Opaque
+data:
+  rootpasswd: cGFzc3dvcmQ=
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mysqldb
+  labels:
+    app: mysqldb
+spec:
+  ports:
+  - port: 3306
+    name: mysql
+  selector:
+    app: mysqldb
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
afaik today there is no easy way to bypass the sidecar at reviews(v3) specific to a version of ratings(v2). Right off the bat there wouldn't be a way to test(e2e) this out right now?

implementation wise there would be 2 versions of the bookinfo-ratings-(x|y).yaml. One where the mysql is loaded with annotation to skip sidecar and one with sidecar enabled.